jenkins/ci.suse.de/cloud-mkcloud9-suse.yaml

```diff
@@ -0,0 +1,58 @@
+- job:
+    name: 'cloud-mkcloud9-suse-x86_64'
+    project-type: multijob
+    node: cloud-trigger
+
+    triggers:
+      - timed: 'H H * * *'
+
+    logrotate:
+      numToKeep: -1
+      daysToKeep: 7
+
+    builders:
+      - multijob:
+          name: 'Standard Gate Checks (non-HA/OVS+Linuxbridge)'
+          condition: SUCCESSFUL
+          projects:
+            - name: openstack-mkcloud
+              node-label: cloud-trigger
+              predefined-parameters: |
+                cloudsource=susecloud
+                TESTHEAD=
+                want_test_updates=1
+                nodenumber=2
+                mkcloudtarget=all_batch
+                scenario=cloud9-2nodes-default.yml
+                want_node_aliases=controller=1,compute-kvm=1
+                label=openstack-mkcloud-SLE12-x86_64
+                job_name=susecloud9: OVS
+            - name: openstack-mkcloud
+              node-label: cloud-trigger
+              predefined-parameters: |
+                cloudsource=susecloud9
+                TESTHEAD=
+                want_test_updates=1
+                nodenumber=4
+                networkingplugin=linuxbridge
+                mkcloudtarget=all
+                label=openstack-mkcloud-SLE12-x86_64
+                job_name=susecloud9: linuxbridge
+      - multijob:
+          name: 'Extra Gate Checks (HA)'
+          condition: SUCCESSFUL
+          projects:
+            - name: openstack-mkcloud
+              node-label: cloud-trigger
+              predefined-parameters: |
+                TESTHEAD=
```

Comment:
Maybe add a FIXME comment here so we remember switching to TESTHEAD=1 when the staging project is ready.

Reply:
no, it is intentionally unset (TESTHEAD has no meaning for cloudsource=susecloudX other than including "test updates" which we don't want here..)